core/jobs/transforms/validation/base_validation.py

```diff
@@ -46,16 +46,12 @@
 MAX_CLOCK_SKEW_SECS = datetime.timedelta(seconds=1)
 
 
-# TODO(#14419): Change naming style of Enum class from SCREAMING_SNAKE_CASE
-# to PascalCase and its values to UPPER_CASE. Because we want to be consistent
-# throughout the codebase according to the coding style guide.
-# https://github.com/oppia/oppia/wiki/Coding-style-guide
-class VALIDATION_MODES(enum.Enum): # pylint: disable=invalid-name
+class ValidationModes(enum.Enum): # pylint: disable=invalid-name
```

Comment:
Remove the pylint disable, ditto elsewhere.

Reply:
@vojtechjelinek Done! Anything else?